docs/lahman.md

```diff
@@ -3,7 +3,7 @@
 Pull data from [Sean Lahman's database](http://www.seanlahman.com/baseball-archive/statistics/), also hosted by [Chadwick Bureau on GitHub](https://github.com/chadwickbureau/baseballdatabank) -- our new source -- using the following functions:
 
 ```python
-from pybaseball.lahman import *
+from pybaseball.datasources.lahman import *
```

Comment:
I don't love this indirection, especially because lahman is user-facing (which `datasources.lahman` isn't)

Reply:
yes - pycharm IDE did this when I moved the file and I didn't catch it